pkg/machinery/go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/jsimonetti/rtnetlink v1.4.0
 	github.com/mdlayher/ethtool v0.1.0
 	github.com/opencontainers/runtime-spec v1.1.0-rc.1
+	github.com/santhosh-tekuri/jsonschema/v5 v5.3.1
```

Comment:
is this supposed to be here?

Reply:
this is because I uncommented the json schema tests

Reply:
ah, it's probably part of the test...

Reply:
I'm always looking to keep the machinery deps to the absolute minimum, but I believe Go should not pull it in as it's only in the test

Reply:
My IDE complained about it, let me remove and test it via our build mechanism.

Reply:
ah ok then, that's my understanding as well.